src/toCamelCase.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+function toCamelCase(str) {
+  return str.replace(/-([a-z])/g, (match, letter) => letter.toUpperCase());
```

Comment:
The regular expression used in the replace function correctly targets hyphenated characters and converts them to uppercase. However, it may not handle cases with multiple consecutive hyphens correctly (e.g., 'font--size' should ideally not be converted but this regex will convert it to 'fontSize'). If such cases are not expected, the current solution is fine.